src/main/java/jp/skypencil/errorprone/slf4j/FormatShouldBeConst.java

```diff
@@ -39,14 +36,16 @@ public Description matchMethodInvocation(MethodInvocationTree tree, VisitorState
             ? 1
             : 0;
 
-    if (IS_CONST.matches(tree.getArguments().get(formatIndex), state)) {
```

Comment:
this did not cause any problems but for consistency i've adjusted it to use `ASTHelpers.constValue` as well, even though this might mean a slight behavior change.
the existing tests continue to pass and the newly added ones show somewhat sane results, so hoping this is good include.